packages/account-v2/src/containers/DocumentSelection/index.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { useBreakpoint } from '@deriv/quill-design';
+import { StandaloneChevronRightRegularIcon } from '@deriv/quill-icons';
```

Comment:
@niloofar-deriv could you please confirm if its ok to import icons directly from @deriv/quill-icons? or do we need to import it from @deriv/quill-icons/Standalone at a reduced size?
i remember seeing your post related this few weeks back. does that apply even now?